protocol/messenger.go

```diff
@@ -477,6 +477,10 @@ func NewMessenger(
 		managerOptions = append(managerOptions, communities.WithCommunityTokensService(c.communityTokensService))
 	}
 
+	if c.collectiblesManager != nil {
+		managerOptions = append(managerOptions, communities.WithCollectiblesManager(c.collectiblesManager))
```

Comment:
Won't it override the option from above? :thinking: 
```
	if walletAPI != nil {
		managerOptions = append(managerOptions, communities.WithCollectiblesManager(walletAPI))
	}
```

Reply:
I think the priority should be `c.collectiblesManager`, but condition can be more strickted:
```
if c.collectiblesManager != nil {
		managerOptions = append(managerOptions, communities.WithCollectiblesManager(c.collectiblesManager))
} else if walletAPI != nil {
		managerOptions = append(managerOptions, communities.WithCollectiblesManager(walletAPI))
}
```

Reply:
improved: https://github.com/status-im/status-go/pull/4737/commits/c840c831cbe2053ce8d7fb612809f61119b16db6

Reply:
https://github.com/status-im/status-go/issues/4786